src/m365/planner/commands/plan/plan-details-get.ts

```diff
@@ -18,7 +18,7 @@ interface Options extends GlobalOptions {
   ownerGroupName?: string;
 }
 
-class PlannerPlanDetailsGetCommand extends GraphCommand {
+class PlannerPlanDetailsGetCommand extends GraphItemsListCommand<any> {
```

Comment:
If we know what kind of items we'll be retrieving, let's not use `any` but specify the exact type to avoid runtime errors.

Reply:
I will bear it in mind next time.